shell/client/setup-wizard/wizard.html

```diff
@@ -44,7 +44,7 @@
                                    mayJumpTo=(mayJumpTo "identity")
                                    route="setupWizardIdentity"
                                    className=itemClassName}}
-      Login provider(s)
+      Identity providers
```

Comment:
I'm OK with either. Also happy to go back to each of the users and ask them about these specific strings (to see which one makes more sense to them) along with a screenshot for context.